CHANGELOG.md

```diff
@@ -1,15 +1,39 @@
 # Changelog
 
+## [1.0.4](https://github.com/chutney-testing/chutney-kotlin-dsl/tree/1.0.4)
```

Comment:
there is compatibility break also in kotlin dsl, so why not to change version to 1.1.0 or 2.0.0 ?